circuitbreaker.py

```diff
@@ -120,18 +125,18 @@ def state(self):
     @property
     def open_until(self):
         """
-        The datetime, when the circuit breaker will try to recover
+        The approximate datetime when the circuit breaker will try to recover
         :return: datetime
         """
-        return self._opened + timedelta(seconds=self._recovery_timeout)
+        return datetime.utcnow() + timedelta(seconds=self.open_remaining)
 
     @property
     def open_remaining(self):
         """
         Number of seconds remaining, the circuit breaker stays in OPEN state
-        :return: int
+        :return: float
         """
-        return (self.open_until - datetime.utcnow()).total_seconds()
+        return (self._opened + self._recovery_timeout) - monotonic()
```

Comment:
Thanks for the change, Thomas! Can't we keep this an `int`? this is still the number of seconds, right?

Reply:
There, ceil() if the value is positive, floor() if it's negative - all tests pass.  Seem reasonable?

Reply:
Great, thanks a lot for the quick adjustment! Very much appreciated, Thomas!